pkgs/by-name/vp/vpkedit/package.nix

```
@@ -0,0 +1,64 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  xcb-util-cursor,
+  xcbutilkeysyms,
+  xcbutilwm,
+  gtkd,
+  openssl,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vpkedit";
+  version = "4.4.2";
+
+  src = fetchurl {
+    url = "https://github.com/craftablescience/ppa/raw/refs/heads/main/debian/package-VPKEdit_${finalAttrs.version}_amd64.deb";
+    hash = "sha256-Fs8h/teSdI2kQzA8xcLDGIFhbq8RC7Z5KOKaA+78Plc=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    xcb-util-cursor
+    xcbutilkeysyms
+    xcbutilwm
+    gtkd
+    openssl
+  ];
+
+  buildInputs = [ dpkg ];
+
+  dontUnpack = true;
+  dontBuild = true;
+  dontStrip = true;
+
+  installPhase = ''
```

its good practice to run the {pre,post}Install hooks incase someone modifies the derivation
```suggestion
  installPhase = ''
    runHook preInstall
```